docs/product/issues/issue-details/index.mdx

```diff
@@ -88,23 +107,34 @@ You can set your own <PlatformLink to="/enriching-events/breadcrumbs/">breadcrum
 
-## Tabs
+These event tags can be filtered by one of the available categories, and clicking any of the hover dropdowns will provide you with a few more search options. Sentry will display tags containing '`.`' characters as a tree structure to make things a bit more readable, but searching should always use the full tag name (e.g. `organization.slug` instead of `slug`).
 
-![Issue Tabs](./img/issue-tabs.png)
+The tags displayed in the main section of the page are specific to the event that you're viewing. The tags displayed in the right-hand sidebar are a summary of all tag values for all events included in the issue. You can set your own tags to make them more useful for debugging as described in <PlatformLink to="/enriching-events/tags/">Customize Tags</PlatformLink>.
 
-This page displays the "Details" tab when it first opens, but several other tabs are available depending on issue category:
+## Screenshot
 
-- **Activity** - A history of activity for the event where team members can share comments.
-- **User Feedback** - Any feedback collected from users through the <PlatformLink to="/user-feedback/">SDK's user feedback</PlatformLink>.
-- **<PlatformLink to="/enriching-events/attachments/">Attachments</PlatformLink>** - Stored additional files, such as config, log or screenshot files that are related to the error event.
-- **Tags** - More detailed information about the tags across all the events included in the issue.
-- **All Events** - A list of all the error events included in the issue.
-- **Merged Issues** - A list of error issues that have been merged with this one.
-- **Similar Issues** - A list of error issues that have a similar stack trace [that you can merge](/concepts/data-management/event-grouping/merging-issues/).
-- **Replays** – A list of [replays](/product/session-replay/) tied to error events included in the issue.
+Sentry provides the ability to take a screenshot and include it as an attachment when a user experiences an error, an exception or a crash. Screenshot attachments sent as a part of an event are displayed in this section. This feature only applies to SDKs with a user interface. It's supported for the following SDKs:
+
+- [.NET Xamarin](/platforms/dotnet/guides/xamarin/)
+- [Android](/platforms/android/enriching-events/screenshots/)
+- [Flutter](/platforms/flutter/enriching-events/screenshots/)
+- [iOS](/platforms/apple/guides/ios/enriching-events/screenshots/)
+- [JavaScript Electron](/platforms/javascript/guides/electron/enriching-events/screenshots/)
+- [React Native](/platforms/react-native/enriching-events/screenshots/)
+- [Unity](/platforms/unity/enriching-events/screenshots/)
+
+## Contexts
+
+![Context Section](./img/issue-context-section.png)
+
+Contexts contain related key/value items (not just strings) that you've set manually in your application, or have been automatically added when Sentry processes an event. Contexts are meant to provide you with related sets of data to help with your debugging.
```

Comment:
```suggestion
Context contains related key/value items (not just strings) that you've set manually in your application or have been automatically added by Sentry. Contexts are meant to provide you with related sets of data to help with your debugging.
```

Reply:
Done manually with updating screenshots and other references to 'Contexts'